spec/support/bin/bundle

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
```

Comment:
Do we need this file?

Reply:
No.  It's a side effect of running tests manually locally.  I've deleted and added to .gitignore.